pkg/providers/providersutils.go

```diff
@@ -277,8 +282,34 @@ func GetCurrentAzSubscriptionId() []string {
 		log.Fatal(err)
 	}
 
-	var ids []string
-	json.Unmarshal(out, &ids)
+	var id []string
```

Comment:
💡 Just a minor suggestion if this works:

```*.go

        var id string
	if err := json.Unmarshal(out, &id); err != nil {
		log.Fatalf("Failed to unmarshal JSON output: %v", err)
	}
```

Reply:
This can be used on 306 as well